cmake/external_mkldnn.cmake

```diff
@@ -82,7 +82,7 @@ foreach(LIB ${MKL_LIBS})
 endforeach()
 
 set(MKLDNN_GIT_REPO_URL https://github.com/intel/mkl-dnn)
-set(MKLDNN_GIT_TAG "0e7ca73")
+set(MKLDNN_GIT_TAG "64e03a1")
```

Comment:
v0.16 is out. We might want to jump to that and skip v0.15

Reply:
sure, make sense.